packages/usa-banner/src/usa-banner.component.js

```diff
@@ -0,0 +1,250 @@
+          action: "Así es como usted puede verificarlo",
+        },
+        domain: {
+          heading: "Los sitios web oficiales usan",
+          text1: "Un sitio web",
+          text2:
+            "pertenece a una organización oficial del Gobierno de Estados Unidos.",
+        },
+        https: {
+          heading1: "Los sitios web seguros",
+          heading2: "usan HTTPS",
+          text1: "Un <strong>candado</strong>",
+          text2:
+            "o <strong>https://</strong> significa que usted se conectó de forma segura a un sitio web",
+          text3:
+            "Comparta información sensible sólo en sitios web oficiales y seguros.",
+        },
+      },
+    };
+  }
+
+  // Get English or Spanish strings. Default to English if an unknown `lang` is passed.
+  // Ex: <usa-banner lang="zy"></usa-banner>
+  get _bannerText() {
+    const content = this.data[this.lang] || this.data.en;
+    return content;
+  }
+
+  // Get the action text and use for both mobile & desktop buttons.
+  get _actionText() {
+    const bannerActionText = this.querySelector('[slot="banner-action"]');
+
+    return bannerActionText?.textContent;
+  }
+
+  domainTemplate(tld) {
+    const { domain } = this._bannerText;
+
+    return html`
+      <img
+        class="usa-banner__icon usa-media-block__img"
+        src="${iconDotGov}"
+        role="img"
+        alt=""
+        aria-hidden="true"
+      />
+      <div class="usa-media-block__body">
+        <p>
+          <strong>
+            <slot name="domain-heading"> ${domain.heading} .${tld} </slot>
+          </strong>
+          <br />
+          <slot name="domain-text">
+            ${domain.text1} <strong>.${tld}</strong> ${domain.text2}
+          </slot>
+        </p>
+      </div>
+    `;
+  }
+
+  static lockIcon() {
+    return html`
+      <span
+        class="usa-banner__icon-lock"
+        role="img"
+        aria-label="Locked padlock icon"
+        part="lock-icon"
+      ></span>
+    `;
+  }
+
+  httpsTemplate(tld) {
+    const { https } = this._bannerText;
+
+    return html`
+      <img
+        class="usa-banner__icon usa-media-block__img"
+        src="${iconHttps}"
+        role="img"
+        alt=""
+        aria-hidden="true"
+      />
+      <div class="usa-media-block__body">
+        <p>
+          <strong>
+            <slot name="https-heading">
+              ${https.heading1} .${tld} ${https.heading2}
+            </slot> </strong
+          ><br />
+          <slot name="https-text">
+            ${unsafeHTML(https.text1)} (${UsaBanner.lockIcon()})
+            ${unsafeHTML(https.text2)} .${tld} ${https.text3}
+          </slot>
+        </p>
+      </div>
+    `;
+  }
+
+  static styles = [unsafeCSS(bannerStyles), unsafeCSS(bannerComponentStyles)];
+
+  render() {
+    const classes = { "usa-banner__header--expanded": this.isOpen };
+    // ? Is there a better way to fallback to a default value is passed value doesn't match?
```

Comment:
```suggestion
    // ? Is there a better way to fallback to a default value if passed value doesn't match?
```

Reply:
for what it's worth, I think your ternary operator below is appropriate for now.